app/styles/_top.scss

```diff
@@ -217,18 +215,17 @@ div.outer-container {
                       }
 
                       nav.nav-area {
-
+                          margin: 0 auto;
+                          width: 800px;
                           font-size: 18px;
                           font-family: LB;
-                          position: fixed;
                           text-align: center;
                           text-transform: uppercase;
                           padding-top: 20px;
-                          display: inline-block;
-                          @media screen and (max-width: $break-small) {
+                          @media screen and (max-width: $break-medium) {
                               padding-bottom: 30px;
                               padding-top: 15px;
-                              background-image: url("./images/nav-bg.png");
+                              background-image: url("/images/nav-bg.png");
```

Comment:
Removing the dot will make `grunt serve` show all images. I updated `deploytos3.bash` accordingly so hopefully it won't affect in production.